### PR TITLE
Add activation parameter at the ClientLibrary Section to specify which activation need to be enabled for the logic yaml

### PR DESCRIPTION
**Pull request by vin-huang**
…e logic YAML.

### CODE REVIEW

**Review by jichangjichang — COMMENTED** [review]
Original behavior should be fine that forcing to test certain activation if user didn't specify any activation type.

**Comment by vin-huang** [comment]
> Original behavior should be fine that forcing to test certain activation if user didn't specify any activation type.

Let user can specify which activation to run in the LibraryClient section

Example: this can have the output for enable gelu, leakrelu and relu. and none
for no activation be enabled.
  LibraryClient:
    - ActivationArgs:
      - [Enum: none]
      - [Enum: gelu]
      - [Enum: leakyrelu]
      - [Enum: relu]

File: tensilelite/Tensile/ClientWriter.py [hunk 1]
```diff
@@ -111,7 +111,13 @@ def main( config ):
     functionNames.append("tensile_%s" % (problemType))
     problemSizes = ProblemSizesMock(exactLogic)
     biasTypeArgs   = BiasTypeArgs(problemType, [problemType["DataType"]])
-    activationArgs = ActivationArgs(problemType, [[{'Enum': 'relu'}]]) if problemType["ActivationType"] == 'all' else ""
+    activationEnums = [[{'Enum': 'none'}]]
```
Comment (jichangjichang) [inline tensilelite/Tensile/ClientWriter.py:114]: keep relu for user not defined case

**Comment by jichangjichang** [comment]
correct the description of  PR and commit

**Comment by vin-huang** [comment]
> correct the description of PR and commit

done

File: tensilelite/Tensile/ClientWriter.py (hunk 1 above)
Comment (vin-huang) [reply tensilelite/Tensile/ClientWriter.py]: done

File: tensilelite/Tensile/ClientWriter.py [hunk 2]
```diff
@@ -111,7 +111,13 @@ def main( config ):
     functionNames.append("tensile_%s" % (problemType))
     problemSizes = ProblemSizesMock(exactLogic)
     biasTypeArgs   = BiasTypeArgs(problemType, [problemType["DataType"]])
-    activationArgs = ActivationArgs(problemType, [[{'Enum': 'relu'}]]) if problemType["ActivationType"] == 'all' else ""
+    activationEnums = [[{'Enum': 'relu'}]]
```
Comment (jichangjichang) [inline tensilelite/Tensile/ClientWriter.py:114]: Could you add your example here to let user know how to set different activation type in LibraryClient stage?

**Review by jichangjichang — APPROVED** [review]
Please merge it AFTER release/rocm-rel-5.7 is created.

**Pull request merged**